lightning/src/ln/functional_tests.rs

```diff
@@ -58,9 +58,12 @@ use ln::chan_utils::CommitmentTransaction;
 #[test]
 fn test_insane_channel_opens() {
 	// Stand up a network of 2 nodes
+	use ln::channel::TOTAL_BITCOIN_SUPPLY_SATOSHIS;
+	let mut cfg = UserConfig::default();
+	cfg.peer_channel_config_limits.max_funding_satoshis = TOTAL_BITCOIN_SUPPLY_SATOSHIS + 1;
```

Comment:
Is there any need to use `TOTAL_BITCOIN_SUPPLY_SATOSHIS` here? Seems strange to use an invalid value when any valid value would suffice.

Reply:
It allows us to cover both the cases of `config.max_funding_satoshis` and the `TOTAL_BITCOIN_SUPPLY` check without recreating the channels with a new config. If it were lower, we'd never get to the `TOTAL_BITCOIN_SUPPLY` check I think